docs/cudf/source/io-supported-types.rst

```diff
@@ -5,58 +5,62 @@ The following table lists are compatible cudf types for each supported IO format
-    | timedelta64[us]       | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ❌     | ❌     | ✅     | ✅     | ❌     | ❌     | ✅     | ✅     |
-    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
-    | timedelta64[ns]       | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ❌     | ❌     | ✅     | ✅     | ❌     | ❌     | ✅     | ✅     |
-    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
-    | datetime64[s]         | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ❌     | ✅     | ✅     |
-    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
-    | datetime64[ms]        | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ❌     | ✅     | ✅     |
-    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
-    | datetime64[us]        | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ❌     | ✅     | ✅     |
-    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
-    | datetime64[ns]        | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ❌     | ✅     | ✅     |
-    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
-    | struct                | ❌     | ❌     | ✅     | ✅     | ❌     | ❌     | ❌     | ❌     | ❌     | ✅     | ✅     | ❌     | ❌     | ✅     | ✅     |
-    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
-    | decimal64             | ❌     | ❌     | ✅     | ✅     | ❌     | ❌     | ❌     | ✅     | ❌     | ❌     | ❌     | ❌     | ❌     | ❌     | ❌     |
-    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    +-----------------------+--------+--------+--------+--------+---------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+
+    |                       |       CSV       |      Parquet    |       JSON       |       ORC       |  AVRO  |       HDF       |       DLPack    |      Feather    |
```

Comment:
HDF reader / writer and Feather reader / writer aren't GPU accelerated either

Reply:
I think the note can be "Not GPU accelerated" so it can apply to any component.

Reply:
Updated for HDF and feather too